client/main.go

```diff
@@ -29,40 +38,61 @@ func New(facility string) (cacher.CacherClient, error) {
 		return fmt.Sprintf("%s:%d", strings.TrimSuffix(addrs[0].Target, "."), addrs[0].Port), nil
 	}
 
-	certURL := os.Getenv("CACHER_CERT_URL")
-	if certURL == "" {
-		auth, err := lookupAuthority("http", facility)
+	var securityOption grpc.DialOption
+
+	useTLS := env.Bool("CACHER_USE_TLS", true)
+
+	// Must be string to check whether or not it has been defined.
+	verifyTLS := env.Get("CACHER_TLS_VERIFY")
+
+	if !useTLS {
+		securityOption = grpc.WithInsecure()
+	} else if verifyTLS != "" {
+		creds := credentials.NewTLS(&tls.Config{
+			InsecureSkipVerify: !isTrue(verifyTLS),
+		})
+		securityOption = grpc.WithTransportCredentials(creds)
```

Comment:
Ok so to make me feel a little better, how about instead of exists - true/false, we do CACHER_OVERRIDE_TLS_VERIFY: true/false, if true: do the InsecureSkipVerify: CACHER_TLS_OVERRIDE?

This is obviously more effort, but I think its fine for something like this, and it would end up being clearer in the code I think too. And lets us make use of `env.Bool`

Reply:
I don't think I completely follow. It seems this work may have been done prematurely and should perhaps be discussed further when we're ready to support these features.